src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedService.java

```diff
@@ -78,15 +79,24 @@ public void processQuestionnaireLinked(
         messageTimestamp);
   }
 
-  private void logErrorIfQidIsLinkedToAnotherCase(
-      UacDTO uac, UacQidLink uacQidLink, String newLinkedCaseId) {
+  private void logUnlinkedCaseEventAgainstOriginalCaseIfQidIsNowLinkedToNewCase(
+      String questionnaireId,
```

Comment:
I suppose the questionnaireId is passed in the `uac` object, so you could lose this parameter if you wanted to save a bit of duplication.